lib/orogen/templates/config/Base.cmake

```diff
@@ -10,6 +10,21 @@ add_custom_command(
     COMMENT "oroGen specification file changed. Run make regen first."
     COMMAND /bin/false)
 
+OPTION(OROGEN_USE_CXX11 "Compile package using the C++11 standard" ON)
```

Comment:
Keep current behaviour unchanged. Set it to OFF by default and then turn it ON explicitely.